### ISSUE TITLE
Multiple instances fight over `user_settings.json`

### ISSUE THREAD

**Issue by borekb** [mike-ward/Markdown-Edit#84]
Tried to run MDE in multiple instances at the same time and this popped up in one of them:

![image](https://cloud.githubusercontent.com/assets/101152/12009708/5125224a-ac84-11e5-9800-1fb2c3b23c6a.png)

**Comment by mike-ward** [comment]
Does this always happen? I run multiples all the time without issue so I'm looking for any additional details to help find this.

**Comment by borekb** [comment]
It was the first time I saw this.

**Comment by mike-ward** [comment]
I've been thinking about removing the file watcher from the settings file (the others, template, snippets would remain). It results in some unexpected behavior. For example, with two instances open change to show/hide line numbers in one of them. Both change. While it's convienent to have the app update when editing the settings file directly, it's perhaps not done often enough to warrant monitoring for changes.

**Comment by cyotek** [comment]
I can also confirm this issue exists, but I wasn't able to find a consistent repro. I got it when I installed MDE for the first time, opened two instances and probably changed settings in both. As I couldn't repro it afterwards, on a whim I deleted the json file, then started two instances again and managed to reproduce it that time. However, I don't have a concrete set of steps. It does seem to involve opening two instances, then activating the option to load the json file in an external editor (urg, Notepad), making a change in the external editor and saving it. I can't remember now if I then tried changing settings again in the two instances, I did a lot of random clicking trying to get a repro ;)
